rakelib/bootstrap.rake

```diff
@@ -1,2 +1 @@
-
-task "bootstrap" => [ "vendor:all", "compile:all" ]
+task "bootstrap" => [ "vendor:gems", "vendor:all", "compile:all" ]
```

Comment:
not useless because I removed the all from the gems task. https://github.com/elastic/logstash/pull/3319/files#diff-77ee63fa86421d6bcb23e149dc0f8854L138